bin/sra_ids_to_runinfo.py

```diff
@@ -8,14 +8,26 @@
 import requests
 import argparse
 
-
 ## Example ids supported by this script
 SRA_IDS = ['PRJNA63463', 'SAMN00765663', 'SRA023522', 'SRP003255', 'SRR390278', 'SRS282569', 'SRX111814']
```

Comment:
I though handling of external samples has moved completely to the `fetchngs` pipeline? Shouldn't this script move there as well?

Reply:
Fixed https://github.com/nf-core/rnaseq/pull/660/commits/57f3a4760ebd78fd4252458f3f8088d6bef5c2c6